include/swift/AST/Module.h

```diff
@@ -357,6 +360,10 @@ class ModuleDecl
     ModuleABIName = name;
   }
 
+  /// Retrieve the underlying name of the alias (if any) used for this module.
+  /// If no module alias is set, it returns getName().
+  Identifier getUnderlyingName() const;
```

Comment:
So if I understand this correctly, if you pass `-module-alias A=B`, then `ModuleDecl::getName()` will return `A` and `ModuleDecl::getUnderlyingName()` will return `B`? It would be helpful if you illustrated this with an example like that, because when I read this doc comment, I don't feel like I'm 100% sure that's what you're saying.

Reply:
sg, will update

Reply:
Thinking about this a little more, I actually think part of what I'm finding confusing is the word "underlying". It sounds like something related to overlays (where the terminology is already unfortunately quite overloaded); if I think about it, I can see which of the two names in an alias would be "underlying" the other, but it's not instantly obvious. Is there a better word you can use instead, like "original" or "actual"?

Reply:
Updated with "real" :) Lmk if that also seems confusing.